pydantic_ai_slim/pydantic_ai/_output.py

```diff
@@ -977,6 +977,14 @@ def __init__(
         self.max_retries = max_retries
         self.output_validators = output_validators or []
 
+    @property
+    def id(self) -> str | None:
+        return '<output>'
```

Comment:
This should have the agent name in it so multiple agents can be used in a workflow